src/bundle/Controller/LocationController.php

```diff
@@ -354,6 +354,11 @@ public function trashAction(Request $request): Response
             if ($result instanceof Response) {
                 return $result;
             }
+
+            $data = $form->getData();
+            $location = $data->getLocation();
+
+            return $this->redirectToLocation($location);
```

Comment:
What if `$location` is null?

Reply:
Is it at all possible for `$location` to be empty when operating on location?

but of course, I can add 'if' if it doesn't find a location then move it to the trash as it is now

Reply:
Ok never mind I checked and it's possible. Added condition to check if the location is not null